examples/ccs811_simpletest.py

```diff
@@ -1,20 +1,18 @@
 import time
-
-from board import SCL, SDA
+import board
 import busio
-
 import adafruit_ccs811
 
-i2c_bus = busio.I2C(SCL, SDA)
-
-ccs = adafruit_ccs811.CCS811(i2c_bus)
+i2c_bus = busio.I2C(board.SCL, board.SDA)
```

Comment:
`i2c_bus` or just `i2c`?

Reply:
I had `i2c`. Must have pasted the wrong update at some point in going back and forth between code.py and simpletest.